sections/results.tex

```diff
@@ -0,0 +1,16 @@
+\section{Results}
+
+The experiment results are presented as a chart\ref{fig:Results} containing the amount of 
+points each agent was able to get while playing against the other agents. The agents are
+RandomAgent, GreedyAgent, RLAgent.
+
+\begin{figure}[h]
+	\includegraphics[width=\linewidth]{figures/chart}
+	\caption{Experiment results. On the x-axis are the agents being evaluated and on the y-axis the average amount of points earned during the experiments.}
```

Comment:
Could we add info about which player is first? i.e. whether it's the one noted on the x-axis or the one indicated by the bar's colour.